### DISCUSSION TITLE
Deleting duplicate file and merging to main branch

### DISCUSSION THREAD

**Discussion by MSM7437** [KadenKercheval/group7#5] [General]
I uploaded a file a second time after making some changes in VSCode, so now I have 2 almost identical files in my branch. How do I delete that? My webpage is completed, at least in VS Code but I havent tested it in GitHub yet.

**Comment by KadenKercheval** [comment]
![image](https://github.com/KadenKercheval/group7/assets/143275579/d7c6e7e2-b768-45b1-934d-ffcd5bf86f68)

**Comment by KadenKercheval** [comment]
Once you click on the file, in the top right there is a 3 button icon and from there you select delete file and the commit the changes.

**Comment by KadenKercheval** [comment]
![image](https://github.com/KadenKercheval/group7/assets/143275579/70a89ef6-f087-4fb5-ae14-e8e1037986e7)

**Comment by KadenKercheval** [comment]
Once you are ready to merge your stuff with the main branch, click on the contribute button and click open pull request.
